### PR TITLE
changed more screen text

### PR DESCRIPTION
**Pull request by noalecohen**

### CODE REVIEW

**Comment by noalecohen** [comment]
finished first task

File: src/AppNavigator.tsx
Comment (BenMoveo) [inline src/AppNavigator.tsx]: please don't push commented code.

File: src/assets/icons/AccountIcon.tsx
Comment (BenMoveo) [inline src/assets/icons/AccountIcon.tsx]: please note the SVG attributes in react becomes camelCase
i.e attributes like stroke-width and fill-rule become strokeWidth and fillRule, etc.

File: src/features/More/MoreScreen.tsx
Comment (BenMoveo) [inline src/features/More/MoreScreen.tsx]: what is this test ?

**Review by BenMoveo — APPROVED** [review]
Ive approved your PR.
Please address my comments (it can be with a comment or a like emoji)
After addressing my comments you can merge your PR.
On the PR homepage at the bottom, you will see a green merge button

File: src/features/More/MoreScreen.tsx
Comment (noalecohen) [reply src/features/More/MoreScreen.tsx]: Sorry, I forgot to delete this check.
I deleted it now.

**Pull request merged**
